honeybee_radiance/cli/octree.py

```diff
@@ -58,8 +66,14 @@ def create_octree_from_folder(
             try:
                 aperture_files = model_folder.aperture_files()
                 scene_files += aperture_files
-            except FileNotFoundError:
+            except Exception:
                 pass  # no apertures available in the model
+        if black_groups:
+            try:
+                group_files = model_folder.aperture_group_files_black()
+                scene_files += group_files
+            except Exception:
+                pass  # no aperture groups available in the model
```

Comment:
Hey @chriswmackey, Having a generic Exception/pass is a really dangerous pattern. Why do you need to do this? Is there a better and more explicit way to handle this? There are papers and articles that say why you should never do such a thing.